packages/@aws-cdk-testing/framework-integ/test/aws-ec2/test/integ.vpc-flow-logs.ts

```diff
@@ -118,13 +118,26 @@ class TestStack extends Stack {
   }
 }
 
+class TransitGatewayFlowLogStack extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const transitGateway = new CfnTransitGateway(this, 'TransitGateway', {});
+    new FlowLog(this, 'FlowLogsCW', {
+      resourceType: FlowLogResourceType.fromTransitGatewayId(transitGateway.ref),
+      flowLogName: 'TransitGatewayFlowLogName',
+    });
+  }
```

Comment:
Is there a reason you did not add a `fromTransitGatewayAttachmentId` resource type FlowLog into the test as well?

Reply:
I worked on it!